paramiko_jump/client.py

```diff
@@ -102,35 +103,34 @@ def _auth(
         )
 
     def connect(
-            self,
-            hostname,
-            port=SSH_PORT,
-            username=None,
-            password=None,
-            pkey=None,
-            key_filename=None,
-            timeout=None,
-            allow_agent=True,
-            look_for_keys=True,
-            compress=False,
-            sock=None,
-            gss_auth=False,
-            gss_kex=False,
-            gss_deleg_creds=True,
-            gss_host=None,
-            banner_timeout=None,
-            auth_timeout=None,
-            gss_trust_dns=True,
-            passphrase=None,
-            disabled_algorithms=None,
+        self,
```

Comment:
Please don't do this -- it's breaking pylint. These are intentionally formatted the way they were. Is this black doing this?

Reply:
Hmm black must have messed it up. I ran black before pushing.